tests/Integration/DataAccess/DoctrineDonationRepositoryTest.php

```diff
@@ -106,18 +101,18 @@ public function testWhenEntityDoesNotExist_getEntityReturnsNull() {
 	}
 
 	public function testWhenDoctrineThrowsException_domainExceptionIsThrown() {
-		$repository = new DoctrineDonationRepository( $this->newEntityManagerThatThrowsOnFind() );
+		$repository = new DoctrineDonationRepository( $this->newEntityManagerThatThrows() );
 
 		$this->expectException( GetDonationException::class );
 		$repository->getDonationById( 42 );
 	}
 
-	private function newEntityManagerThatThrowsOnFind(): EntityManager {
+	private function newEntityManagerThatThrows(): EntityManager {
 		$entityManager = $this->getMockBuilder( EntityManager::class )
 			->disableOriginalConstructor()->getMock();
 
 		$entityManager->expects( $this->any() )
-			->method( 'find' )
+			->method( $this->anything() )
```

Comment:
This is cool - can avoid binding to whatever method the implementation happens to be using